src/AWSCore.jl

```diff
@@ -202,11 +190,12 @@ Service endpoint URL for `request`.
 function service_url(aws, request)
     endpoint = get(request, :endpoint, request[:service])
     region = "." * aws[:region]
+
     if endpoint == "iam" || (endpoint == "sdb" && region == ".us-east-1")
         region = ""
     end
-    string("https://", endpoint, region, ".amazonaws.com",
-           request[:resource])
+
+    return string("https://", endpoint, region, ".amazonaws.com", request[:resource])
```

Comment:
If you remove the "." then you can do:
```suggestion
    return "https://$(endpoint).$(region).amazonaws.com$(request[:resource])"
```